README.md

```diff
@@ -160,7 +160,7 @@ If you are willing to contribute, open a pull request to complete, update or fil
 
 ### V
 
-* [V8](/glossary/V8.md): *to be completed.*
+* [V8](/glossary/V8.md): V8 is Google's open source javaScript engine. It's what Chrome is running, but it's also used for other projects like Node.js and MongoDB.
```

Comment:
Could we get rid of `V8 is` at the beginning of the sentence?
Also would be super could if we could use smart quotes (`’`) in place of dumb quotes. :)